### PR TITLE
Refactoring CmsValidator (internal CRL resolution)

### PR DESCRIPTION
**Pull request by lucassaldanha**
## PR description

Loading CRL list inside CmsValidator (instead of externally)

## Changelog

- [X] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

File: pki/src/main/java/org/hyperledger/besu/pki/cms/CmsValidator.java [hunk 1]
```diff
@@ -178,4 +178,18 @@ private boolean isCertificateTrusted(
       throw new RuntimeException("Error validating certificate chain", e);
     }
   }
+
+  private Optional<CertStore> loadCRLs(final KeyStoreWrapper truststore) {
```
Comment (jframe) [inline pki/src/main/java/org/hyperledger/besu/pki/cms/CmsValidator.java:182]: This is really loading CRL as such, maybe createCertStore?

Comment (jframe) [inline pki/src/main/java/org/hyperledger/besu/pki/cms/CmsValidator.java:182]: Noticed there is CRLUtil that does something similar to this function, should that be used instead or this replaces that one?

Comment (lucassaldanha) [reply pki/src/main/java/org/hyperledger/besu/pki/cms/CmsValidator.java:182]: CRLUtil is used when loading the CRL pem file. Not for the truststore. They are similar but not the same.

**Review by jframe — APPROVED** [review]
LGTM

**Pull request merged**